src/unix/udp.c

```diff
@@ -202,6 +255,12 @@ static int uv__udp_recvmmsg(uv_udp_t* handle, uv_buf_t* buf) {
         flags |= UV_UDP_PARTIAL;
 
       chunk_buf = uv_buf_init(iov[k].iov_base, iov[k].iov_len);
+#if defined(__linux__)
+      if (flag & MSG_ERRQUEUE &&
+          uv__udp_recvmsg_errqueue(handle, &msgs[k].msg_hdr, &chunk_buf,
+                                   (const struct sockaddr*) &peers[k], flags))
+        continue;
```

Comment:
```suggestion
      if ((flag & MSG_ERRQUEUE) &&
          uv__udp_recvmsg_errqueue(handle, &msgs[k].msg_hdr, &chunk_buf,
                                   (const struct sockaddr*) &peers[k], flags)) {
        continue;
      }
```
Braces for legibility and parens around the & expression because otherwise some compilers complain.

(I disagree with compilers on that subject because the precedence is unambiguous but warnings are annoying.)